<launch>

  <!-- Turtlesim Launcher -->
  <node name = "sim" pkg = "turtlesim" type = "turtlesim_node" output = "screen"/>
  
  <!-- Node Launcher with saturation args -->
  <node name = "ex2_turtlecontrol" pkg = "exercise2" type = "ex2_turtlecontrol" output = "screen">
  <param name = "vel_lin_min" value = "$(arg vel_lin_min)"/>
  <param name = "vel_lin_max" value = "$(arg vel_lin_max)"/>
  <param name = "vel_ang_min" value = "$(arg vel_ang_min)"/>
  <param name = "vel_ang_max" value = "$(arg vel_ang_max)"/>
  </node>
  
   <!-- Commands Publisher Launcher --> 
  <node name="topic_pub" pkg="rostopic" type="rostopic" args="pub -r 10 /control exercise2/control &quot; lin_vel: 1.0 &#13; ang_vel: 2.0 &quot;" output="screen"/>
  
  <!-- Rqt Plot Launcher -->	
  <node name = "rqt_plot" pkg = "rqt_plot" type = "rqt_plot" output = "screen" args =" /turtle1/cmd_vel/angular/z /turtle1/cmd_vel/linear/x /control">

</launch>
